### PR TITLE
Cq 4351085

### PR DESCRIPTION
**Pull request by barshat7**
## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/adaptive-form/pageheader/.content.xml [hunk 1]
```diff
@@ -1,129 +1,122 @@
...
                     <component
-                            jcr:primaryType="nt:unstructured"
-                            sling:resourceType="core-components-examples/components/demo/component">
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="core-components-examples/components/demo/component">
                         <container
-                                jcr:primaryType="nt:unstructured"
-                                sling:resourceType="core-components-examples/components/form/container">
+                            jcr:primaryType="nt:unstructured"
+                            sling:resourceType="core-components-examples/components/form/container">
```
Comment (nit23uec) [inline examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/adaptive-form/pageheader/.content.xml:81]: FYI : @amit-mnnit is fixing this part where RT of form should point to our component. without that, our runtime clientlibs are not loaded.

**Review by nit23uec — APPROVED** [review]

**Comment by nit23uec** [comment]
@barshat7 , pls take care of copyright headers. new files should have 2023.

File: examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/clientlibs/forms-clientlib-site/styles/components/adaptive-form/accordion.less [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .cmp-accordion {
-  .panelContainer();
-  margin-right: @field-right-margin;
+  //.panelContainer();
```
Comment (nit23uec) [inline examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/clientlibs/forms-clientlib-site/styles/components/adaptive-form/accordion.less:2]: why this is commented?
cc @amit-mnnit

File: examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/adaptive-form/pagefooter/.content.xml [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="forms-components-examples/components/demo/component">
+            <guideContainer
+              fd:version="2.1"
+              fieldType="form"
+              jcr:primaryType="nt:unstructured"
+              sling:resourceType="forms-components-examples/components/form/container">
+              <container
+                jcr:primaryType="nt:unstructured"
+                fieldType="demo-component"
```
Comment (nit23uec) [inline examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/adaptive-form/pagefooter/.content.xml:100]: why fieldType is necessary?

File: examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/clientlibs/forms-clientlib-site/styles/components/adaptive-form/accordion.less (hunk 2 above)
Comment (barshat7) [reply examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/clientlibs/forms-clientlib-site/styles/components/adaptive-form/accordion.less:2]: accordion's css classes are messing up with site's css and breaking the example page

File: examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/adaptive-form/pagefooter/.content.xml (hunk 3 above)
Comment (barshat7) [reply examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/adaptive-form/pagefooter/.content.xml:100]: Field Type is a required field in Forms core components, or else it breaks.

Comment (rismehta) [reply examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/adaptive-form/pagefooter/.content.xml:100]: This is not required @barshat7. It should not break

**Pull request merged**